CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Removing metrics configuration
```

Comment:
Can this be updated to make mention of this being removed as it is not part of trace GA

Reply:
Fixed